src/main/java/com/example/automator/helper/ABMRunner.java

```diff
@@ -36,13 +36,23 @@ public static ArrayList<Double> runABM(Parameters parameters) {
             workspace.command(String.format("set percentage_negative_WoM %s", parameters.getPercentageNegativeWoM()));
             workspace.command(String.format("set base_adoption_probability %s", parameters.getBaseAdoptionProbability()));
 
+            
+            
+            // SET INTERVENTION PARAMETERS
+            workspace.command("set direct_ad_type \"Direct Ad\"");
+            workspace.command("set direct_ad_frequency 365");
+            workspace.command("set train_chiefs_frequency 0");
+            workspace.command("set max_budget 10000");
+            workspace.command("set direct_ad_nr_of_villages 50");
+            workspace.command("set percentage_of_villagers_addressed 50");
+            workspace.command("set train_chiefs_nr 50");
```

Comment:
Das ist ja jetzt noch hardcoded und die Werte müssten noch mit den input parametern aus dem GET call ersetzt werden. Aber könne wir auch in einem extra Branch machen (da ja nicht Teil vom Optimizer).

Reply:
Ah ja das hab ich zum ausprobieren mal hardcoded. Die Parameter soll der User ja dann im Frontend anpassen können.
So wie ich das verstehe, sollen die Werte vom Frontend Input in die Parameter Klasse einfliessen und dann via Getter abgerufen werden können oder? Hab jedoch gerade keine Vorstellung davon, wie ich auf den Frontend Input zugreifen kann. 😅